config/config.go

```diff
@@ -289,12 +309,19 @@ func NewNetworkAgentConfig(yamlPath string) (*AgentConfig, error) {
 	}
 
 	if util.PathExists(yamlPath) {
-		if err := cfg.loadNetworkYamlConfig(yamlPath); err != nil {
+		config.Datadog.AddConfigPath(yamlPath)
```

Comment:
should we turn this segment and the one above into a function since it's used twice ?